### PR TITLE
added a media query for screen sizes less than 400px

### PR DESCRIPTION
**Pull request by VeritasX**
Added a media query in the _layout Sass file that looks at the site_title class and changes the font size when  the screen is at or below 400 pixels

### CODE REVIEW

**Comment by DemonWav** [comment]
55 files changes. Most of it garbage. Fix your setup and only commit the files that are actually changed.

**Comment by VeritasX** [comment]
I changed the files and got rid of the line endings

File: _sass/_layout.scss [hunk 1]
```diff
@@ -191,3 +191,9 @@ body:before {
 .page-heading {
     font-size: 20px;
 }
+
+@media(max-width:400px){
+    .site-title{
+        font-size:16px;
+    }
+}
```
Comment (DemonWav) [inline _sass/_layout.scss]: Please leave a newline at the end of the file.

Comment (VeritasX) [inline _sass/_layout.scss]: will do

Comment (VeritasX) [inline _sass/_layout.scss]: done let me know

**Pull request merged**